GOVERNANCE.md

```diff
@@ -35,6 +35,12 @@ mailing list containing the following information:
 
 Two other maintainers need to second the nomination. If no one objects in 5 working days (U.S.), the nomination is accepted.  If anyone objects or wants more information, the maintainers discuss and usually come to a consensus (within the 5 working days). If issues can't be resolved, there's a simple majority vote among current maintainers.
 
+## Maintainer duties
+
+We have only one required activity in which maintainers have to engage from time to time: releasing new versions of Jaeger! This process ideally takes only a couple of hours, but requires coordination on different fronts. Even though the process is well documented, it is not without eventual glitches, so, each release needs a "Release Manager". How it works is described in the [RELEASE.md](RELEASE.md) file.
```

Comment:
"Only one duty" is not accurate, maintainers are expected to review PRs or otherwise participate.

Reply:
Added a paragraph before that to reflect that. My original intent was to say that the only concrete task we require them to do is release. All others are things we want maintainers to do, but we don't actively ask them to, except perhaps for a few PRs where we tag people specifically based on their knowledge of the affected code.

Anyway, hopefully it's better now.